lib/cli/run-helpers.js

```diff
@@ -143,10 +143,8 @@ exports.handleFiles = ({
     try {
       newFiles = utils.lookupFiles(arg, extension, recursive);
     } catch (err) {
-      if (err.message.indexOf('cannot resolve path') === 0) {
-        console.error(
-          `Warning: Could not find any test files matching pattern: ${arg}`
-        );
+      if (err.code === 'ERR_MOCHA_NO_FILES_MATCH_PATTERN') {
```

Comment:
future PR: use variables instead of hardcoded strings

Reply:
Regrettably, it's the standard Node error code processing pattern.
[This](https://github.com/cswl/nodejs-error-codes) seems more like what I would have preferred -- exported constants.

Reply:
Yep, I had it like that on first iteration of PR but the change was lost in the ether. Will look at updating.